File: app/src/main/java/vng/zalo/tdtai/zalo/zalo/models/UserInfo.java
package vng.zalo.tdtai.zalo.zalo.models;

import com.google.firebase.Timestamp;

public class UserInfo {
    public String phone;
    public String avatar;
    public Timestamp birthDate;
    public boolean isMale;
    public Timestamp joinDate;

    public UserInfo(){}

    @Override
    public String toString() {
        return "{phone: "+phone+", avatar: "+avatar+", birthDate: "+birthDate+"}";
    }
}

File: app/src/main/java/vng/zalo/tdtai/zalo/models/UserInfo.kt
package vng.zalo.tdtai.zalo.models

import com.google.firebase.Timestamp

data class UserInfo(
        var phone: String? = null,// also id
        var avatarUrl: String? = null,
        var birthDate: Timestamp? = null,
        var isMale: Boolean? = false,
        var joinDate: Timestamp? = null)